src/api/app/models/workflow/step/errors.rb

```diff
@@ -0,0 +1,17 @@
+module Workflow::Step::Errors
+  extend ActiveSupport::Concern
+
+  class CanNotLinkPackageNotFound < APIError
```

Comment:
shouldn't this class name end with `Error` ?

Reply:
Having these two files as an example, it seems that's not the tendency:

https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/workflow/errors.rb
https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/branch_package/errors.rb

Reply:
Having `Errors` in the module is already enough. It results in `Workflow::Step::Errors::CanNotLinkPackageNotFound`.

Reply:
it isn't used anymore right?

Reply:
Yes, we have removed this file.